### PR TITLE
Grails6  and security plugins 6.2.0 upgrade

### PR DESCRIPTION
**Pull request by qifeng-bai**

### CODE REVIEW

File: buildSrc/build.gradle [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    maven { url "https://nexus.ala.org.au/content/groups/public/" }
+    maven { url "https://repo.grails.org/grails/core" }
+    mavenCentral()
+}
+dependencies {
+    implementation "org.grails:grails-gradle-plugin:6.0.0"
+    implementation "org.grails.plugins:hibernate5:7.3.0"
+    implementation "gradle.plugin.com.github.erdi.webdriver-binaries:webdriver-binaries-gradle-plugin:2.6"
+    implementation "com.bertramlabs.plugins:asset-pipeline-gradle:3.4.6"
+}
```
Comment (adam-collins) [inline buildSrc/build.gradle:12]: Delete this if it is not used.

File: gradle.properties [hunk 2]
```diff
@@ -1,7 +1,10 @@
-grailsVersion=5.2.1
-gorm.version=7.3.2
+grailsVersion=6.0.0
+grailsGradlePluginVersion=6.0.0
+gormVersion=8.0.0
```
Comment (adam-collins) [inline gradle.properties:3]: hibernate5 dependency that is imported does not match the new gormVersion. Was that tested? This differs from documentation https://gorm.grails.org/latest/hibernate/manual/

**Review by adam-collins — CHANGES_REQUESTED** [review]
```
Reason: Task ':war' uses this output of task ':compileGroovyPages' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed. Please refer to https://docs.gradle.org/7.6.2/userguide/validation_problems.html#implicit_dependency for more details about this problem.
```

The above build problem is reported and should be fixed.

File: grails-app/conf/application.yml [hunk 3]
```diff
@@ -244,9 +259,12 @@ security:
 webservice:
     jwt: true
     jwt-scopes: "openid users/read"
+    webservice:
+        client-id: "Replace it"
+        client-secret: "Replace it"
```
[the remark this answers is not in the document]
Comment (qifeng-bai) [reply grails-app/conf/application.yml:264]: Fixed

**Pull request closed without merging**